### DISCUSSION TITLE
Profile readme.md section for organisations

### DISCUSSION THREAD

**Discussion by darshan-jain** [community/community#22755] [New to GitHub]
<p>Have you ever come across an organisation and you have no clue what the organisation does?<br>
Well, don’t worry you are not the only one stuck there. Github had launched an option for users to greet profile visitors with some insights about the repo owner. I genuinely think this can be highly useful for open source repositories of organisations so that it would provide good insight about the entire repo and what sort of open source contribution they would like to see.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/darshan-jain">@darshan-jain</a>,</p>
<p>Welcome to the community, we are happy you are here!</p>
<p>Thanks for sharing, here is a handy Open-Source Project with tips to make your organizations README awesome :sparkler:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/readmeTips/readmeTips.github.io" target="_blank" rel="noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/20666190/181073025-1cd68992-2880-4b7e-baa6-a76222e0204f.png" class="thumbnail onebox-avatar" width="420" height="420">

<h3><a href="https://github.com/readmeTips/readmeTips.github.io" target="_blank" rel="noopener">readmeTips/readmeTips.github.io</a></h3>

<p>Tips for building a good README file on GitHub or anywhere - readmeTips/readmeTips.github.io</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>In the meantime, we’re looking forward to seeing you around!</p>
<p>AG</p>

**Comment by joostschouppe** [comment]
<p>I want to solve this exact problem for my organisation, however it seems like this neat trick (as <a href="https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-profile/managing-your-profile-readme" rel="noopener nofollow ugc">documented here</a>) only works for individual accounts, and not for organisation accounts. But maybe I’m missing something, because I don’t see anything explicitely stating it is impossible</p>
